<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMatchPage</class>
 <widget class="QWidget" name="AdminMatchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Матчи (Администратор)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Экспорт в CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>Введите название клуба или турнира</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="inputFieldsLayout">
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string>Дата матча:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="homeClubLabel">
         <property name="text">
          <string>Домашний клуб:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="homeClubComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="awayClubLabel">
         <property name="text">
          <string>Гостевой клуб:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="awayClubComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="scoreLabel">
         <property name="text">
          <string>Счет:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scoreEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="tournamentLabel">
         <property name="text">
          <string>Турнир:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tournamentComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>